{"name": "登陆失败-用户名格式错误", "status": "passed", "attachments": [{"name": "stderr", "source": "0788a10e-3fa3-48c1-9657-d162e93445ec-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'id': 7, 'title': '登陆失败-用户名格式错误', 'method': 'post', 'url': 'member/login', 'request_data': '{\"mobile_phone\":\"1550000000\",\"pwd\":\"123456789\"}', 'expected': None, 'check_sql': None}"}], "start": 1636643789149, "stop": 1636643789427, "uuid": "4983c83a-6d60-4a5d-9525-0570b3380914", "historyId": "bf39c435278d016e28fddd7d0f10a83d", "testCaseId": "44f56b03f0866dee9fbf282cd6e7f5f6", "fullName": "testcases.test_2_login_api.TestLogin#test_login", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('req_assert')"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_2_login_api"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-04EC16A3"}, {"name": "thread", "value": "9236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_2_login_api"}]}